data/json/items/gun/nail.json

```diff
@@ -1,5 +1,29 @@
 [
   {
+    "id": "coilgun",
+    "copy-from": "gun_base",
+    "type": "GUN",
+    "name": "coilgun",
+    "description": "A homemade gun, using electromagnets to accelerate a ferromagnetic projectile to high velocity.  Powered by UPS.",
+    "weight": 3341,
+    "volume": 11,
+    "price": 75000, "//": "Hard to make, plentiful and cheap ammo, and silent - people will want this thing.",
+    "bashing": 10,
+    "to_hit": -2,
+    "material": "steel",
+    "skill": "rifle",
+    "ammo": "nail",
+    "ranged_damage": 8,
```

Comment:
Changed stats as requested.

However, it should probably cause more damage than nail rifle because the projectile is faster, not just have better armor piercing.